python/xorbits/_mars/dataframe/base/string_.py

```diff
@@ -91,13 +96,30 @@ def execute(cls, ctx, op):
         return _string_method_to_handlers[op.method].execute(ctx, op)
 
 
-class SeriesStringMethodBaseHandler:
+class StringMethodBaseHandler:
     @classmethod
     def call(cls, op, inp):
-        empty_series = build_empty_series(inp.dtype)
-        dtype = getattr(empty_series.str, op.method)(
+        empty_data = None
+        type_d = None
```

Comment:
Would it be better to remove `type_d`? The meaning of this var name is not clear and it seems removing it makes the code short and more readable.

Reply:
resolved in the latest version